website/project/metadata/schemas.py

```diff
@@ -53,7 +54,8 @@
     'Real World Evidence in Health Outcomes Minimum Recommended Form',
     'Qualitative Preregistration',
     'ASIST Results Registration',
-    'ASIST Hypothesis/Capability Registration'
+    'ASIST Hypothesis/Capability Registration',
+    'Null Hypothesis Significance Testing (NHST)'
```

Comment:
no need to update this list -- this was only used for ordering output on the old flow for making a registration

Reply:
IIRC this is actually needed so this schema is read during `ensure_schemas`.